### PR TITLE
Monty hall facelift

### PR DESCRIPTION
**Pull request by jwendin**
Convert react code to functional components.
Use hooks for getting backend health
Bump spring version
Bump java version
Bump all frontend dependencies

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Ett förberett projekt baserat på create-react-app och spring initializr.
 #### Systemkrav
 
 [Maven](https://maven.apache.org/)<br>
-[Java8](http://www.oracle.com/technetwork/java/javase/overview/index.html)<br>
+[Java11](http://www.oracle.com/technetwork/java/javase/overview/index.html)<br>
```
Comment (gabagoo) [inline README.md]: Urlen borde kanske peka på openJDK? https://openjdk.java.net/projects/jdk/11/

File: backend/pom.xml [hunk 2]
```diff
@@ -14,14 +14,14 @@
...
-		<version>1.5.10.RELEASE</version>
+		<version>2.3.3.RELEASE</version>
 		<relativePath/> <!-- lookup parent from repository -->
 	</parent>
 
 	<properties>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-		<java.version>1.8</java.version>
+		<java.version>1.11</java.version>
```
Comment (gabagoo) [inline backend/pom.xml]: 1.11 -> 11

Comment (jwendin) [reply backend/pom.xml]: ah fattar

File: README.md (hunk 1 above)
Comment (jwendin) [reply README.md]: Ja det fixar jag

**Pull request merged**